projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -133,3 +133,10 @@
 		margin-top: 0 !important;
 	}
 }
+
+:where(.wp-block-jetpack-subscriptions input[type="email"]) {
```

Comment:
We tend to use :where in core, I don't know if it's the right decision for jetpack, but my instinct says yes